specs/destination.go

```diff
@@ -19,9 +19,9 @@ type Destination struct {
 }
 
 const (
-	WriteModeAppend WriteMode = iota
+	WriteModeOverwriteDeletestale WriteMode = iota
 	WriteModeOverwrite
```

Comment:
Does `WriteModeOverwrite` translates to `upsert` (which is the current default)?